frontend/src/App.tsx

```diff
@@ -1,27 +1,31 @@
 import React from "react";
-import { Button, Container, Row } from "react-bootstrap";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 
-const App: React.FC = () => {
-	const lettersAndNumerics = "abcdefghijklmnopqrstuvwxyz0123456789",
-		letters: String[] = lettersAndNumerics.split("");
+import HomePage from "./components/HomePage/HomePage";
+import Navigation from "./components/Navigation/Navigation";
+import About from "./components/About/About";
+import Contact from "./components/Contact/Contact";
+import Books from "./containers/Books/Books";
 
+const App: React.FC = () => {
 	return (
-		<div className='App'>
-			<Container>
-				<h1>HonnoMushi Test</h1>
-				<Row>
-					{letters.map((letter, index) => (
-						<Button
-							key={index}
-							className='col-2 m-4 p-3'
-							variant='outline-success'
-						>
-							{letter}
-						</Button>
-					))}
-				</Row>
-			</Container>
-		</div>
+		<Router>
```

Comment:
Remove the router, this adds another <Router> to the original <Router> parent which causes the bugs